tests/test_zarr.py

```diff
@@ -420,15 +418,7 @@ def test_write_zarr_multiscale(
     data = [
         da.from_zarr(uri, component=str(i)) for i in range(len(zgroup.data))
     ]
-    assert len(data) == 3
-
-    image = data[0][0, :, :].compute()  # convert dask array to numpy array
-
-    for d in data:
-        assert (
-            np.linalg.norm(image - d[0, :, :].compute()) == 0
-        )  # validate against the same method from scikit-image
-        image = downscale_local_mean(image, (2, 2)).astype(np.uint8)
+    assert len(data) == 3  # 3 layers
```

Comment:
I like the original test. The problem here is the `empty` camera. `random` or `sin` should be fine. If the test time is a concern we can reduce the number of frames.

Reply:
The original test is actually wrong, because it's only comparing against 1 downsampled frame, but the Zarr writer is averaging two downsampled frames. I can add it back in with a fix in a separate PR, but it's holding up bug fixes in acquire-common and the Zarr driver that need to go out.

Reply:
My $0.02:  Get the test to pass in a more limited context (1 down-sampled frame instead of 2)?   Then add an issue to produce a more comprehensive test after release.